contrib/test_jenkins.sh

```diff
@@ -275,6 +275,11 @@ build_cuda() {
             $MAKE clean
             $MAKE
             $MAKE distclean
+
+            ../contrib/configure-release --prefix=$ucx_inst --with-cuda --with-gdrcopy
+            $MAKE clean
+            $MAKE
+            $MAKE distclean
             module unload dev/gdrcopy
         else
             ../contrib/configure-devel --prefix=$ucx_inst --with-cuda
```

Comment:
Does it make sense to add release build for CUDA without gdrcopy configuration?

Reply:
yes, @bureddy 
but we can merge this PR w/o this for now

Reply:
currently, without gdrcopy is complete subset of the with gdrcopy configuration.  anyway, will enable it in next set of fixes